### ISSUE TITLE
Document screenshot format

### ISSUE THREAD

**Issue by Zero3** [MachinePublishers/jBrowserDriver#236]
I'm just playing around with this library for the first time, and I noticed that there is no documentation anywhere (as far as I can tell) about the format of the bytes returned from calling `driver.getScreenshotAs(OutputType.BYTES)`. I'm guessing it's a PNG file from searching the net, but it would be nice if this was documented. I noticed that the API is entangled in Selenium interfaces, so I don't know if this is something you can fix in jBrowserDriver, or it is an upstream issue.

**Comment by hollingsworthd** [comment]
Yes, it's PNG. The JavaDocs get inherited from the Selenium project. Maybe there's a way to inherit them and supplement with a custom comment.
